v22.1/aws-dms.md

```diff
@@ -178,6 +187,15 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 
     Try selecting **Full LOB mode** in your [task settings](#step-2-2-task-settings). If this does not resolve the error, select **Limited LOB mode** and gradually increase the **Maximum LOB size** until the error goes away. For more information about LOB (large binary object) modes, see the [AWS documentation](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.LOBSupport.html).
 
+- If you encounter an `ABORT_REASON_CLIENT_REJECT` error when migrating an especially large table, and are running v22.1.14 or later, set the following session variable using [`ALTER ROLE ... SET {session variable}`](alter-role.html#set-default-session-variable-values-for-a-role):
```

Comment:
We might want to define what we mean by large here.  Also, can we provide more information as to where we'd expect customers to see that error.  Sometimes it shows up in the detailed cloud flare logging.  Sometimes you need to dig into the CRDB logs to find it.

Reply:
@otan Can you help me define "large" here? I guessed it was on the order of millions of rows.

Reply:
Yeah millions is a good approximation